### ISSUE TITLE
Cancel Order Confirmation page needs vocalization prompt for users to review items to be cancelled

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13610]
**Describe the bug**
- When focusing on the "Submit Request" button an alternative description should prompt the user to review the items about to be cancelled.
- This information is already on the page, but it needs to be associated with the button as a way to prevent errors.
- Since we have process buttons on top and bottom of the table, the message _"The following items will be included in the cancellation request."_ needs adjustment for screen reader users. E.g _"Items **on this page** will be included in the cancellation request"_.

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**

1. Open any Screen Reader (VO, NVDA, JAWS).
2. Add any product to the cart.
3. Checkout the product.
4. Go to Order History and click on the order number.
5. In the Order Details page, scroll down, click "Cancel Items" button.
6. In the Cancel Order Page, add an item to cancel, press "Continue".
7. Focus on  "Submit Request" button
8. Hear absence of any narration advising the user about what will happen after pressing the button 
9. After at the button is display (hear the absence of vocalization).

**Expected behavior**
- Preventive message gets vocalized when screen reader user focuses the "Submit Request" button.

**Screenshots**
Potential solution:

![Screen Shot 2021-09-02 at 7 12 02 PM](https://user-images.githubusercontent.com/15113219/131930888-34ea6d46-0587-48b8-bc6a-12348c14047d.png)

![Screen Shot 2021-09-02 at 7 13 03 PM](https://user-images.githubusercontent.com/15113219/131930947-24e7150d-665c-4ea7-9f3b-e1179e83a903.png)

**Comment by saad-mo** [comment]
@developpeurweb The paragraphs are CMS-driven (`CMSParagraphComponent`) and do not support any attributes like id or classes. I will continue to investigate further, but is there any other workaround for this?

**Comment by saad-mo** [comment]
Update: it is possible to expose `id` for each `CMSParagraphComponent`, but it's a global breaking change that might need blamed review.

**Comment by saad-mo** [comment]
Need arch review/input

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

Notice that it only reads the message on top and although it does not read the exact same words, it translates to the same thing "Items on this page will be included in the cancellation request". 
But it does not read the message at the bottom "Please note that it may take a few moments for your order .... "

@developpeurweb Is it good enough for you? Or should it read the message at the bottom too? 


<img width="1201" alt="Screen Shot 2022-02-23 at 1 12 06 PM" src="https://user-images.githubusercontent.com/45409515/155381877-fbcfb66c-d85a-4540-99fd-24f0774de5b8.png">

After submitting the cancellation, it's ok.
![cancelOrder1](https://user-images.githubusercontent.com/45409515/155380817-ac99d2dc-73f6-4e33-8923-9f11ed10a508.png)

**Comment by developpeurweb** [comment]
@KateChuen this is OK, and is good enough for **error prevention** (user submitting without knowing consequences), and confirmation provides enough feedback on the outcome.
We excluded the last paragraph as it made the narration too verbose and didn't really provide additional value, since the confirmation already says the order "will be" updated.
